Task 66&67

MonomythConvergence opened the pull request.

MonomythConvergence committed:
not ready yet, commiting to stash changes before pull

MonomythConvergence committed:
Merge branch 'develop' of github.com:Student2506/practicum-android-diploma into task_66&67

# Conflicts:
#	feature-filter/src/main/res/values/dimens.xml

MonomythConvergence committed:
not ready yet, commiting to stash changes before pull

MonomythConvergence committed:
Merge branch 'develop' of github.com:Student2506/practicum-android-diploma into task_66&67

# Conflicts:
#	feature-filter/src/main/res/values/dimens.xml
#	feature-filter/src/main/res/values/styles.xml

MonomythConvergence committed:
ready

Student2506 commented on feature-filter/src/main/java/ru/practicum/android/diploma/filter/profession/presentation/ui/ProfessionFragment.kt:
```diff
@@ -90,24 +98,25 @@ internal class ProfessionFragment : Fragment() {
         binding.selectButton.setOnClickListener {
-            // ❗ pass selectedIndustry to viewmodel here
+            // ❗ todo pass selectedIndustry to filter fragment or viewmodel here or do sharedprefs
+            findNavController().navigateUp()
```
Логику возврата расписать можно тоже...
Допиши тоже, что в у меня PR

MonomythConvergence committed:
Merge branch 'develop' of github.com:Student2506/practicum-android-diploma into task_66&67

MonomythConvergence committed:
1.bundle + navigation to filter
2.Profession -> Industry, не выдержал уже и поменял, слишком режет глаз.

MonomythConvergence committed:
1.bundle + navigation to filter
2.Profession -> Industry, не выдержал уже и поменял, слишком режет глаз.

MonomythConvergence committed:
1.bundle + navigation to filter
2.Profession -> Industry, не выдержал уже и поменял, слишком режет глаз.

+ detektFormat
